From 5cfe3ef88349e34712077c53392c1ed3f2354179 Mon Sep 17 00:00:00 2001
Message-Id: <5cfe3ef88349e34712077c53392c1ed3f2354179.1421886431.git.chang-joon.lee@intel.com>
In-Reply-To: <2a59efb195329ac28200fa6e68e1170a4cf8d03c.1421886431.git.chang-joon.lee@intel.com>
References: <2a59efb195329ac28200fa6e68e1170a4cf8d03c.1421886431.git.chang-joon.lee@intel.com>
From: Deepak S <deepak.s@intel.com>
Date: Fri, 12 Dec 2014 09:09:44 +0530
Subject: [PATCH 11/11] FOR_UPSTREAM [VPG] : drm/i915: Move unnessesary
 DRM_ERROR ("invalid or inactive unpin_work") to
 DRM_DEBUG.

"do_intel_finish_page_flip" is called even when unpin work is null or inactive.
This usually happens when we update the DSPSURF without flip ioctal.
Usually happens when we do modeset. This patch moves error message to
debug to avoid false alarm.

Issue: GMINL-4795
Change-Id: I553cbebb5b737a793cfe9500c935be12956f0607
Signed-off-by: Deepak S <deepak.s@intel.com>
---
 drivers/gpu/drm/i915/intel_display.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/intel_display.c b/drivers/gpu/drm/i915/intel_display.c
index 111cb57..483a40b 100644
--- a/drivers/gpu/drm/i915/intel_display.c
+++ b/drivers/gpu/drm/i915/intel_display.c
@@ -9963,7 +9963,7 @@ static void do_intel_finish_page_flip(struct drm_device *dev,
 
 	if (work == NULL || atomic_read(&work->pending) < INTEL_FLIP_COMPLETE) {
 		spin_unlock_irqrestore(&dev->event_lock, flags);
-		DRM_ERROR("invalid or inactive unpin_work!\n");
+		DRM_DEBUG_DRIVER("invalid or inactive unpin_work!\n");
 		return;
 	}
 
-- 
1.7.9.5

